### PR TITLE
Updated PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by DalilaBA**
I Updated the file

### CODE REVIEW

**Comment by anirudhprabhakaran3** [comment]
This is in reference to https://github.com/publiclab/plots2/issues/9967 I believe? In the future, before working on the PR, please leave a message on the issue so that we know someone is working on it. Will just review the PR now. Thank you!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -8,10 +9,12 @@ Make sure these boxes are checked before your pull request (PR) is ready to be r
...
-> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!
+<!--We're happy to help you get this ready -- don't be afraid to ask for help, and don't be discouraged if your tests fail at first!-->
 
-If tests do fail, click on the red `X` to learn why by reading the logs.
 
-Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 
+<!--If tests do fail, click on the red X to learn why by reading the logs.-->
 
-Thanks!
+<!-- Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
```
Comment (anirudhprabhakaran3) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: I think you need to add a closing bracket here, at the end?

Comment (noi5e) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: @anirudhprabhakaran3 Wow, great catch! Thanks for the code review.

@DalilaBA please go ahead and make the change if you can!

**Comment by noi5e** [comment]
@DalilaBA I just merged a PR that included the missing closing bracket in #9994, but thank you so much for this pull request. Closing this now.

**Pull request closed without merging**